.github/workflows/kyber.yml

```diff
@@ -0,0 +1,101 @@
+name: Extract and TC Kyber
+
+on:
+  pull_request:
+  merge_group:
+  workflow_dispatch:
+  push:
+    branches: [main]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  hax:
+    # if: ${{ github.event_name == 'workflow_dispatch' || github.event_name == 'merge_group' }}
```

Comment:
Do you want to run it only on the merge queue? 🙂 
Also, it only needs 10 minutes and the `Install & test / nix-action` job is slower. So we could run it directly. But I'm fine keeping it on merge queue only.

Reply:
Oh, I forgot to uncomment that indeed... Thanks!
I think it's fast only because hax was cached (in the log that takes less than 1 minute), but otherwise, that will take around 9 minutes of build, so something like 20 minutes in total